### PR TITLE
[skipci] doc: tools-v2/develop.md(add work flow)

### PR DESCRIPTION
**Pull request by Xinlong-Chen**
更改tools-v2开发文档
+ 新增部署过程中遇到一些问题及其解决方法。
+ 新增推荐的调试流程，便于按图索骥。

### CODE REVIEW

**Review by ilixiaocui — APPROVED** [review]

**Comment by ilixiaocui** [comment]
LGTM！

File: tools-v2/docs/zh/develop.md [hunk 1]
```diff
@@ -293,6 +301,19 @@ make
...
 生成的二进制文件保存为 `tools-v2/sbin/curve`。
 
+> 若出现诸如没有 `protoc-gen-go` 、`protoc-gen-go-grpc`，但在上面步骤已经install的情况，可查看 `GOROOT` 下（一般为 `~/go` ）是否有二进制文件，若有，可以将其拷贝至go的安装路径后再重新编译。
+>
+> ```shell
+> cd ~/go/bin
+> cp ./protoc-gen-go* /usr/local/go/bin
+> ```
+>
+> 若出现musl-gcc相关的报错，可进行如下操作直接编译：
```
Comment (Cyber-SiKu) [inline tools-v2/docs/zh/develop.md:311]: You can also give the installation documentation:
https://command-not-found.com/musl-gcc

File: tools-v2/docs/zh/develop.md [hunk 2]
```diff
@@ -293,6 +301,19 @@ make
 
 生成的二进制文件保存为 `tools-v2/sbin/curve`。
 
+> 若出现诸如没有 `protoc-gen-go` 、`protoc-gen-go-grpc`，但在上面步骤已经install的情况，可查看 `GOROOT` 下（一般为 `~/go` ）是否有二进制文件，若有，可以将其拷贝至go的安装路径后再重新编译。
+>
+> ```shell
+> cd ~/go/bin
+> cp ./protoc-gen-go* /usr/local/go/bin
+> ```
```
Comment (Cyber-SiKu) [inline tools-v2/docs/zh/develop.md:309]: You can also add ~/go/ to the environment variable:
export $PATH=$HOME/go/bin:$PATH
or create a soft link

File: tools-v2/docs/zh/develop.md (hunk 1 above)
Comment (Xinlong-Chen) [reply tools-v2/docs/zh/develop.md]: done

File: tools-v2/docs/zh/develop.md (hunk 2 above)
Comment (Xinlong-Chen) [reply tools-v2/docs/zh/develop.md]: done

**Review by Cyber-SiKu — APPROVED** [review]

**Pull request merged**